### PR TITLE
2.0

### PR DESCRIPTION
**Pull request by hypeJunction**
* Restyled
* Now uses action to load notifications and update DOM
* Moved pages to resources
* Added auto updating whenever an ajax call is made to the server

notifier

- [x] Remove height limitation from `notifier/all` page so it won't need vertical scroll
- [ ] Move files and make changes to them in separate commits

### CODE REVIEW

File: views/default/notifier/notifier.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+/* Lightbox */
+.notifier-lightbox-wrapper { width: 570px; min-height: 410px; margin: 20px 15px; }
+
+#notificationstable td.notifiertogglefield { width: 50px; text-align: center; vertical-align: middle; }
+#notificationstable td.notifiertogglefield input { margin-right: 10px; }
+#notificationstable td.notifiertogglefield a { display: inline-block; font-family: 'FontAwesome'; font-size: 18px; line-height: 20px; vertical-align: middle; text-rendering: auto; -webkit-font-smoothing: antialiased; -moz-osx-font-smoothing: grayscale; }
+#notificationstable td.notifiertogglefield a:after { content: "\f0ac"; }
+#notificationstable td.notifiertogglefield a.notifiertoggleOff { color: #cccccc; }
+#notificationstable td.notifiertogglefield a.notifiertoggleOn { color: #4078c0; }
```
Comment (juho-jaakkola) [inline views/default/notifier/notifier.css]: Could you revert this to the original formatting? If possible, also make separate commits for moving the file and modifying contents, so it's easier to see what has changed.

File: views/default/notifier/notifier.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#notifier-popup { padding: 0; background: white; width: 345px; position: absolute; font-size: 14px; line-height: normal; }
+#notifier-popup .elgg-pagination { display: none !important; }
+#notifier-popup .elgg-no-results, #notifier-popup .elgg-ajax-loader { margin: 0; padding: 8px; }
+
+.notifier-none { display: block; text-align: center; padding: 8px; }
+
+.elgg-list-notifier { max-height: 300px; overflow-y: auto; margin: 0; }
```
Comment (juho-jaakkola) [inline views/default/notifier/notifier.css]: Use max-height only if inside `.elgg-module`

**Comment by juho-jaakkola** [comment]
Most changes are ok, but I dislike the fact that some commits both relocate and modify the files. This makes commit history more difficult to read.

**Comment by juho-jaakkola** [comment]
What kind of screen did you use for testing? On mine I can barely see any difference between read and unread items.

**Comment by hypeJunction** [comment]
Won't have any more time to invest into this. I will use my fork for a project, feel free to cherry pick whatever you need.
